showcase/app/components/page-components/form/file-input/sub-sections/containers.gts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import style from 'ember-style-modifier';
+
+import ShwFlex from 'showcase/components/shw/flex';
+import ShwTextH2 from 'showcase/components/shw/text/h2';
+
+import { HdsFormFileInputField } from '@hashicorp/design-system-components/components';
+
+export interface SubSectionContainersSignature {
+  Args: {
+    showHighlight: boolean;
+  };
+}
+
+const SubSectionContainers: TemplateOnlyComponent<SubSectionContainersSignature> =
+  <template>
+    <ShwTextH2>Containers</ShwTextH2>
+
+    <div
+      class="{{if
+          @showHighlight
+          'shw-component-form-file-input-layout-highlight'
+        }}"
+    >
+      <ShwFlex @gap="2rem" as |SF|>
+        <SF.Item
+          @label="Within a fixed-width container"
+          {{style width="220px"}}
```

Comment:
[suggestion/non-blocker] consider using a `<ShwOutliner>` component to wrap the `FileInputField` and apply the fixed width to it, like we do here https://github.com/hashicorp/design-system/blob/main/showcase/app/components/page-components/copy/button/sub-sections/base-elements.gts#L82 (you get the benefit of seeing the actual container and be certain that the `FileInputField`  is not overflowing (and this is also tested for visual regressions)